packages/devtools_app/test/memory/allocation_tracing/allocation_profile_tracing_view_test.dart

```diff
@@ -142,9 +142,9 @@ void main() {
       final refresh = find.text('Refresh');
       expect(refresh, findsOneWidget);
 
-      expect(find.text('Trace'), findsOneWidget);
+      expect(find.text('Trace'), findsNWidgets(2));
```

Comment:
can you add a comment describing why there are two?

Reply:
done